lib/SIL/SILFunctionBuilder.cpp

```diff
@@ -201,7 +209,6 @@ SILFunctionBuilder::getOrCreateFunction(SILLocation loc, SILDeclRef constant,
       // SWIFT_ENABLE_TENSORFLOW
       addFunctionAttributes(F, storage->getAttrs(), mod, constant);
     }
-    // SWIFT_ENABLE_TENSORFLOW
```

Comment:
Note: this comment should be removed because the following line is in parity with master.